cohortextractor/emis_backend.py

```diff
@@ -744,73 +744,51 @@ def _none_of_these_codes_occur_on_same_day(self, joined_table, codelist):
         """
 
     def patients_registered_practice_as_of(self, date, returning=None):
+        # At the moment we can only return current values for the fields in question.
```

Comment:
Does this need a similar check on the date as below?

Reply:
Thanks.